### PR TITLE
finished the rails api gem

### PR DESCRIPTION
**Pull request by triciasfu**
deleted all the assets folders

### CODE REVIEW

**Comment by SamLau95** [comment]
Whoa nice job! :)

File: bluebase_api/lib/bluebase_api/app_builder.rb [hunk 1]
```diff
@@ -89,7 +90,7 @@ def remove_html_files # Added by Quinton and Tricia
       remove_file "public/404.html"
       remove_file "public/422.html"
       remove_file "public/500.html"
-      run "rm favicon.ico"
+      run "rm -rf public/favicon.ico"
```
Comment (SamLau95) [inline bluebase_api/lib/bluebase_api/app_builder.rb:93]: Does `remove_file` not work here?

Comment (triciasfu) [inline bluebase_api/lib/bluebase_api/app_builder.rb:93]: Noo, it doesn't remove the .ico file! I'm not really sure why

File: bluebase_api/lib/bluebase_api/generators/app_generator.rb [hunk 2]
```diff
@@ -33,6 +33,11 @@ def bluebase_api_customization
       invoke :setup_git_and_github
       invoke :setup_heroku_apps
       invoke :outro
+      invoke :customize_app_directory
+      invoke :customize_public_directory
+      invoke :customize_vendor_directory
+      invoke :customize_lib_directory
+      invoke :customize_tmp_directory
```
Comment (SamLau95) [inline bluebase_api/lib/bluebase_api/generators/app_generator.rb:37]: Did you move these below everything else to denote that these were methods you changed? If so, do you think they should stay in this position? (Kinda nitpicky)

Comment (triciasfu) [inline bluebase_api/lib/bluebase_api/generators/app_generator.rb:37]: ok i can go ahead and move them with the rest of the customize commands!

**Comment by SamLau95** [comment]
Also, have you tested making a git repo / heroku apps?

**Comment by SamLau95** [comment]
:+1:

**Pull request merged**